test_clickhouse.troy

```diff
@@ -0,0 +1,52 @@
+define flow clickhouse
```

Comment:
I would put this as an example in the docs, but remove from the repo, I don't think we generally keep examples in the repo.

Reply:
I wrote a similar example in the docs. It was just a file I used when I was experimenting locally. It has been removed in [fb93fb7](https://github.com/tremor-rs/tremor-runtime/pull/1538/commits/fb93fb75452b35cfd008c5586faa7735d1dfdacd).